delta/app/src/main/resources/app.conf

```diff
@@ -199,6 +199,21 @@ app {
 
     # configuration for the persistence of progress of projections
     persist-progress-config = ${app.defaults.persist-progress-config}
+
+    # configuration for the automatic provisioning of projects.
+    automatic-provisioning {
+      enabled = false
+      permissions = ["resources/read", "resources/write", "projects/read"]
+      description = "Auto provisioned project"
+      enabled-realms = {
+        realm = "users-realm"
+      }
+      api-mappings = {
+        documents = "https://bluebrain.github.io/nexus/vocabulary/defaultElasticSearchIndex"
```

Comment:
why is this here? This is part of the default api mappings, it does not need to be explicit.

Reply:
I just put it there as an example config.

Reply:
Are we going to use this in our use case (MOOC)? If not, we don't need it at all.

If it is just an example, you should put it with comments `#`